riotfile.py

```diff
@@ -2773,6 +2773,24 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="langchain",
+            command="pytest {cmdargs} tests/contrib/langchain",
+            pys=select_pys(min_version="3.9"),
+            pkgs={
+                "langchain": ["==0.0.192", latest],
+                "openai": latest,
```

Comment:
dumb question: Do we need to specify all these packages? Are some of these packages installed transitively by openai and langchain?

Reply:
The below packages are all independent (all different LLM provider libraries), though `psutil` and `exceptiongroup` can be removed once riot is upgraded on our end.